.pending/breaking/sdk/4001-Upgrade-to-Tendermint-v0-31-1

```diff
@@ -0,0 +1 @@
+#4001 Upgrade to Tendermint v0.31.1
```

Comment:
This does not look like breaking API/ABI in the SDK. On the other hand, I think we need to mention it under improvements->tendermint: `sdkch add improvements tendermint`.

Reply:
It is breaking though. Users of 0.33 will not be able to use this, correct?

Reply:
I moved the log to tendermint breaking. Look at our changelog -- it's inconsistent. Some versions have it as improvement and some as breaking. I think it should be breaking for the reason above.